File: bp/bp.js
// Node 11.2.0

const fs = require("fs");

function getInput() {
  return fs.readFileSync("input.txt", "utf-8").trim().split("\n");
}

function main() {
  const puzzle = getInput();

  // Code here
}

if (module === require.main) {
  const start = process.hrtime.bigint();
  console.log(main());
  console.log(process.hrtime.bigint() - start);
}

File: bp/bp.py
# Python 3.12.0


import typing


def get_input() -> list[str]:
    with open("input.txt", "r") as f:
        return f.read().split()


def main() -> typing.Any:
    puzzle = get_input()

    # Code here


if __name__ == "__main__":
    import time

    start = time.perf_counter()
    print(main())
    print(time.perf_counter() - start)
